src/components/ArchiveBody/ArchiveBody.css

```diff
@@ -0,0 +1,37 @@
+.ArchiveBody{
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    height: 100%;
+    margin-top: 4rem;
```

Comment:
There's a neat shorthand that will shorten these 3 lines into just one:
`margin: 4rem 0 10rem 3rem` (the positions are `top right bottom left`)

Also, I think we can convert these margin values to pixels. I don't think we would want these margin sizes to get bigger/smaller depending on the font size.